### ISSUE TITLE
Bug: "Go to previoius/next" command opens file in sidebar if file is selected

### ISSUE THREAD

**Issue by greetclammy** [pjeby/quick-explorer#108]
### Steps to replicate
1. Select file (not merely open it) by pressing on it twice or by executing the *Reveal file in navigation* Command Palette command.
<img width="441" alt="Screen Shot 2024-11-18 at 08 35 44" src="https://github.com/user-attachments/assets/deaa6f2e-fd41-43d9-9d5f-044f6e2f8f2d">

2. Use a *Quick Explorer* hotkey to go to previous or next file. I use Cmd+Alt+↑ and Cmd+Alt+↓.

### Expected result
The previous/next file is opened in the main area/editor. The file explorer remains in its place.

### Actual result
The file opens in the sidebar, replacing the file explorer entirely:
<img width="1416" alt="Screen Shot 2024-11-18 at 08 38 44" src="https://github.com/user-attachments/assets/1859e177-972a-4dc8-a3cb-21fa53a2b441">

### Debug info 
```
SYSTEM INFO:
	Obsidian version: v1.7.6
	Installer version: v1.7.5
	Operating system: Darwin Kernel Version 20.6.0: Thu Jul  6 22:12:47 PDT 2023; root:xnu-7195.141.49.702.12~1/RELEASE_X86_64 20.6.0
	Login status: not logged in
	Language: en
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 5
	Plugins enabled: 1
		1: Quick Explorer v0.2.11

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by greetclammy** [comment]
Same issue as #86

**Comment by pjeby** [comment]
Should be fixed in 0.2.12, thanks for the detailed report.
